packages/cli/src/commands/installedschema/delete.ts

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash'
+
+import { flags } from '@oclif/command'
+
+import { InstalledSchemaApp } from '@smartthings/core-sdk'
+
+import {selectAndActOn, APICommand, withLocations} from '@smartthings/cli-lib'
+
+
+export default class InstalledSchemaAppDeleteCommand extends APICommand {
+	static description = 'delete the installed schema connector instance'
+
+	static flags = {
+		...APICommand.flags,
+		'location-id': flags.string({
+			char: 'l',
+			description: 'filter results by location',
+			multiple: true,
+		}),
+		verbose: flags.boolean({
+			description: 'include location name in output',
+			char: 'v',
+		}),
+	}
+	static args = [{
+		name: 'id',
+		description: 'installed schema conntector UUID',
+	}]
+
+	primaryKeyName = 'isaId'
+	sortKeyName = 'appName'
+	protected listTableFieldDefinitions = ['appName', 'partnerName', 'partnerSTConnection', 'isaId']
+
+	async run(): Promise<void> {
+		const { args, argv, flags } = this.parse(InstalledSchemaAppDeleteCommand)
+		await super.setup(args, argv, flags)
+
+		if (this.flags.verbose) {
+			this.listTableFieldDefinitions.splice(3, 0, 'location')
+		}
+
+		let locationIds = flags['location-id']
+
+		await selectAndActOn<InstalledSchemaApp>(this, args.id,
+			async () => {
+				if (!locationIds) {
+					locationIds = (await this.client.locations.list()).map(it => it.locationId)
+				}
+
+				const isas = _.flatten(await Promise.all(locationIds.map(async (locationId) => {
+					try {
+						return (await this.client.schema.installedApps(locationId))
+					} catch(e) {
+						return []
+					}
+				})))
+
+				if (this.flags.verbose) {
+					return await withLocations(this.client, isas)
+				}
+				return isas
+			},
```

Comment:
This function is the same as the one in `installedschema.ts`. I would make it an exported function there and use the same one here.